README.md

```diff
@@ -65,76 +78,65 @@ Info on Zoom's API and webhook functionality can be found at:
     1. Click "Enable" and confirm
     1. The follow-up screen will show the changes being applied. The last one will show a red X instead of a green checkmark.
        Apparently this is fine for our purposes and doesn't cause problems, so safe to ignore.
-1. Populate the Zoom meeting schedule database. See the *Schedule DB* section below for more details.
+1. (Optional) Populate the Zoom meeting schedule database. See the *Schedule DB* section below for more details.
     1. Export the DCE Zoom schedule google spreadsheet to a CSV file
-    1. Run `invoke schedule.import -c [csv file] -s [semester] -y [year]`
+    1. Run `invoke schedule.import-csv [filepath]`
 
 That's it. Your Zoom Ingester is deployed and operational. To see a summary of the
 state of the CloudFormation stack and the Lambda functions run `invoke stack.status`.
 
-#### dependency changes
+### Set up Zoom webhook notifications (Optional)
 
-Dependencies for the project as a whole and the individual functions are managed using
-the `pip-tools` command, `pip-compile`. Top-level dependencies are listed in a `.in` file
-which is then compiled to a "locked" `.txt` version like so:
 
-`pip-compile -o requirements.txt requirements.in`
+Once the Zoom Ingester pipeline is operational you can configure your Zoom account to
+send completed recording notifications to it via the Zoom Webhook settings.
 
-Both the `.in` and `.txt` files are version-controlled, so the initial compile was
-only necessary once. Now we only have to run `pip-compile` in a couple of situations:
+1. Get your ingester's webhook endpoint URL. You can find it in the `invoke stack.status` output
+or by browsing to the release stage of your API Gateway REST api.
+1. Go to [marketplace.zoom.us](marketplace.zoom.us) and log in. Under "Develop" select "Build App."
+1. Give your app a name. Turn off "Intend to publish this app on Zoom Marketplace."
+Choose app type "Webhook only app."
+1. Click "Create." Fill out the rest of the required information,
+and enter the API endpoint under "Event Subscription."
+1. Make sure to subscribe to "All recordings have completed" events.
+1. Activate the app when desired. (For development it's recommended that you only leave the notifications active while you're actively testing.)
 
```

Comment:
Here we need a section on setting up the on-demand ingest. Where to get the on-demand endpoint url and maybe where it goes in the opencast configuration (IDK where that is)